### ISSUE TITLE
GE.1+ ORF1a:P1921S with 2 branches around 20% of Kenyan sequences from 5-15 to 6-09

### ISSUE THREAD

**Issue by Fearuncle** [sars-cov-2-variants/lineage-proposals#200]
![image](https://github.com/sars-cov-2-variants/lineage-proposals/assets/78371872/772f224b-cbfc-49a7-a91a-92996b5a1e1d)
![image](https://github.com/sars-cov-2-variants/lineage-proposals/assets/78371872/ca3e26ba-bf4b-45ce-ada8-4b08c9c8f722)
tree:https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_2fc16_be2830.json?c=gt-nuc_6026&label=id:node_6950739
first seq: 2023-05-17
OR098896.1 OR098919.1 OR125920.1 OR098918.1 OR125918.1 OR125916.1 OR125921.1 OR098915.1 OR125925.1 OR125919.1 OR125927.1 OR098920.1

**Comment by FedeGueli** [comment]
Yeah i've proposed it but as 521P is not real i'll close that issue and leave this open

**Comment by FedeGueli** [comment]
Let it open having emerged in an undersampled area.

**Comment by FedeGueli** [comment]
Designated GE.1.2 ( a step further in the tree from S:148T but it is the same described here)
